.github/workflows/ci.yml

```
@@ -227,26 +227,10 @@ jobs:
             git tag "v${NEW_VERSION}"
             git push origin "v${NEW_VERSION}"
             
-            # Create GitHub Release for features/breaking changes
+            # Create GitHub Release for features/breaking changes  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)